File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/ui/dialog/DialogModule.java
package com.androidcollider.easyfin.common.managers.ui.dialog;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class DialogModule {

    @Provides
    DialogManager provideDialogManager() {
        return new DialogManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/ui/dialog/DialogModule.kt
package com.androidcollider.easyfin.common.managers.ui.dialog

import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class DialogModule {
    @Provides
    fun provideDialogManager(): DialogManager {
        return DialogManager()
    }
}